src/commands/interactive/start-interactive-mode.ts

```diff
@@ -0,0 +1,66 @@
+import { CmdGlobalOptions } from "../cmd-options.ts";
+import { exploreInteractive } from "./untagged-tenants.ts";
+import { listTenantAction, listTenantsCostAction } from "../tenant.command.ts";
+import { Select } from "../../deps.ts";
+import { OutputFormat } from "../../presentation/output-format.ts";
+import { interactiveDate } from "./inputInteractiveDate.ts";
+
+export async function startInteractiveMode(options: CmdGlobalOptions) {
+  console.clear();
+  const interactivehelp =
+    '\n\n\nWelcome to the collie-cli interactive mode. This mode allows you to herd your tenants in a quicker, more userfriendly way.\n\n\n "LIST ALL TENANTS"\nis equivalent to "collie tenant list"\n\n"LIST ALL TENANTS WITH COST"\nis equivalent to "collie tenant costs"\n\n"EXPLORE TENANTS WITH MISSING TAGS"\nis the superpower of the interactive mode.Go check it out!\n\n';
+
+  let running = true;
+  while (running) {
+    const action: string = await Select.prompt({
+      message: "Select what you want to do",
+      options: [
+        { name: "LIST ALL TENANTS", value: "alltenants" },
+        { name: "LIST ALL TENANTS WITH COST", value: "tenantcost" },
+        { name: "EXPLORE TENANTS WITH MISSING TAGS", value: "exploremissing" },
```

Comment:
d: Explore all tenants with missing costs: I understand that in a way that we want to find all tenants that have no tags or that are missing a certain tag.
When I do choose that interactive option the following happens:
 -  I can select to sort them by highest cost or not -> i choose yes
 - I then need to pick dates, which I do.
 
 Afterwards I get prompted to select a tag. 
Please make is clear that I need to select a tag so I can later see tenants that are missing that tag.
Now it would be nice to see a list of tenants already and then be able to interactively choose a tenant from that list when I want to drill down or quit.

Reply:
additionally: This also happens when I choose to not sort them before.

Reply:
as this is not absolutely necessary we can proceed with the PR without and polish later